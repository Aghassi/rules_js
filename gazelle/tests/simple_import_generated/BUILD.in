# Generated .js
genrule(
    name = "generator",
    outs = ["genlib.js"],
    cmd = "echo 'export const ANSWER = 42;' > $@",
)