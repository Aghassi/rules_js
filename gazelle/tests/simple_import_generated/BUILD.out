load("@npm//@bazel/typescript:index.bzl", "ts_project")

# Generated .js
genrule(
    name = "generator",
    outs = ["genlib.js"],
    cmd = "echo 'export const ANSWER = 42;' > $@",
)

ts_project(
    name = "simple_import_generated",
    srcs = ["main.ts"],
    deps = ["//:genlib.js"],
)
